pagopa/io-app
# fix: [IOPID-422,IOPID-435] First onboarding bug/FastLogin what's new

## Short description
This PR:
- fixes a bug on first onboarding
- removes some sentences from fast login what's new
-  fixes a bug on bottom sheet safe area
-  fixes the idp selection screen scrolling

## List of changes proposed in this pull request
- updated tests
- ts/components/IdpsGrid.tsx: modified style to allow full screen expansion.
- ts/features/whatsnew/hook/useWhatsNew.tsx: updated import.
- ts/features/whatsnew/screen/FastLoginWhatsNew.tsx: removed some sentences.
- ts/features/whatsnew/store/reducers/index.ts: updated import.
- ts/sagas/startup.ts: dispatched an action to fix the bug (read the comment).
- ts/screens/authentication/IdpSelectionScreen.tsx: updated screen.
- ts/utils/hooks/bottomSheet.tsx: added safe area margin.

## How to test
- Idp selection screen should now properly scroll on small screens.
- Generate a bottom sheet with a lot of text. it should not invade the safe area.
- Try first on boarding.
